src/lab-tiles/tests-ordered-tile.component.tsx

```diff
@@ -6,12 +6,16 @@ import { useLabTestsStats } from "../summary-tiles/laboratory-summary.resource";
 const ReferredTileComponent = () => {
   const { t } = useTranslation();
 
-  const { count: testOrderedCount } = useLabTestsStats("");
+  const { data } = useLabTestsStats("");
+
+  const count = data?.filter(
```

Comment:
This might have been a mistake. You named the const `count` but are passing filtered data. Does the response return orders or a count?